common/src/main/java/com/duckduckgo/app/global/exception/UncaughtExceptionEntity.kt

```diff
@@ -27,8 +26,8 @@ data class UncaughtExceptionEntity(
     @PrimaryKey(autoGenerate = true) val id: Long = 0,
     val exceptionSource: UncaughtExceptionSource,
     val message: String,
-    val timestamp: Long = System.currentTimeMillis(),
-    val version: String = BuildConfig.VERSION_NAME
+    val version: String,
+    val timestamp: Long = System.currentTimeMillis()
```

Comment:
`BuildConfig.VERSION_NAME` cannot be access from a lib gradle module, so I made `version` required so that the caller side can provide the proper version